lib/qa/authorities/linked_data/find_term.rb

```diff
@@ -240,7 +250,8 @@ def predicates_with_subject_uri(expected_uri) # rubocop:disable Metrics/MethodLe
         end
 
         def append_performance_data(results)
-          performance = { predicate_count: results['predicates'].size,
+          pred_count = results['predicates'].present? ? results['predicates'].size : 0
+          performance = { predicate_count: pred_count,
```

Comment:
This original error was that `results['predicates']` was nil and causing `.size` to raise an exception.  This prevents that.  This really shouldn't be needed, but if it were to happen again in another circumstance, it would fail more gracefully.